cdap-watchdog/src/main/java/co/cask/cdap/logging/gateway/handlers/LogHandler.java

```diff
@@ -91,12 +93,13 @@ public void getRunIdLogs(HttpRequest request, HttpResponder responder, @PathPara
                            @QueryParam("stop") @DefaultValue("-1") long toTimeSecsParam,
                            @QueryParam("escape") @DefaultValue("true") boolean escape,
                            @QueryParam("filter") @DefaultValue("") String filterStr) {
-    LoggingContext loggingContext =
-      LoggingContextHelper.getLoggingContextWithRunId(namespaceId, appId, programId,
-                                                      ProgramType.valueOfCategoryName(programType),
-                                                      runId);
     RunRecordMeta runRecord = programStore.getRun(
       Id.Program.from(namespaceId, appId, ProgramType.valueOfCategoryName(programType), programId), runId);
+
+    ProgramType type = ProgramType.valueOfCategoryName(programType);
```

Comment:
`ProgramType.valueOfCategoryName(programType)` in the preceding line can also be replaced by `type` (calling it `programType` would be better I think).